src/style.css

```
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+  
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  padding: 0;
+}
+
+body {
+  font-size: 16px;
+}
+
+.form {
+  position: relative;
+  justify-content: center;
+}
+
+.input {
+  width: 100%;
+  font-family: "Avenir Heavy", Arial , sans-serif;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+}
+
+.input::placeholder {
+  font-family: "Avenir Book", Arial , sans-serif;
+  font-weight: inherit;
+}
+
+.input-big {
+  position: relative;
+  height: 70px;
+  padding-left: 62px;
+  margin-bottom: 20px;
+  line-height: 70px;
+  font-family: "Avenir Heavy", Arial , sans-serif;
+  font-size: 16px;
```

Роби відступи між різними групами правил, так читається набагато простіше
```suggestion
  position: relative;
  
  height: 70px;
  padding-left: 62px;
  margin-bottom: 20px;
  line-height: 70px;
  
  font-family: "Avenir Heavy", Arial , sans-serif;
  font-size: 16px;
```